deploy/charts/oneblock/templates/service.yaml

```diff
@@ -13,3 +13,18 @@ spec:
       name: https
   selector:
     {{- include "oneblock.selectorLabels" . | nindent 4 }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: oneblock-webhook-server
```

Comment:
Since the service name will kept the same as the variable `webhookServerName` defined in the server.go, it prefer to make it fixed. The name `oneblock-webhook` is better.